tests/trestle/core/models/element_test.py

```diff
@@ -24,32 +24,32 @@
 from trestle.oscal import target
 
 
-def test_element_get_at(sample_target: target.TargetDefinition):
+def test_element_get_at(sample_target_def: target.TargetDefinition):
     """Test element get method."""
-    element = Element(sample_target)
+    element = Element(sample_target_def)
 
     # field alias should succeed
     assert element.get_at(
         ElementPath('target-definition.metadata.last-modified')
-    ) == sample_target.metadata.last_modified
+    ) == sample_target_def.metadata.last_modified
 
     # field name should fail
     assert element.get_at(ElementPath('target-definition.metadata.last_modified')) is None
 
-    assert element.get() == sample_target
+    assert element.get() == sample_target_def
     assert element.get_at() == element.get()
-    assert element.get_at(ElementPath('target-definition.metadata')) == sample_target.metadata
-    assert element.get_at(ElementPath('target-definition.metadata.title')) == sample_target.metadata.title
-    assert element.get_at(ElementPath('target-definition.targets')) == sample_target.targets
-    assert element.get_at(ElementPath('target-definition.targets.*')) == sample_target.targets
-    assert element.get_at(ElementPath('target-definition.metadata.parties.*')) == sample_target.metadata.parties
-    assert element.get_at(ElementPath('target-definition.metadata.parties.0')) == sample_target.metadata.parties[0]
+    assert element.get_at(ElementPath('target-definition.metadata')) == sample_target_def.metadata
+    assert element.get_at(ElementPath('target-definition.metadata.title')) == sample_target_def.metadata.title
+    assert element.get_at(ElementPath('target-definition.targets')) == sample_target_def.targets
+    assert element.get_at(ElementPath('target-definition.targets.*')) == sample_target_def.targets
+    assert element.get_at(ElementPath('target-definition.metadata.parties.*')) == sample_target_def.metadata.parties
+    assert element.get_at(ElementPath('target-definition.metadata.parties.0')) == sample_target_def.metadata.parties[0]
     assert element.get_at(ElementPath('target-definition.metadata.parties.0.uuid')
-                          ) == sample_target.metadata.parties[0].uuid
+                          ) == sample_target_def.metadata.parties[0].uuid
```

Comment:
Going forward I would suggest that we use the 800-53 base catalog as a reference object of OSCAL. The reason why I specifically suggest this is that the 800-53 object provided by NIST they will be responsible for updating to reflect the latest object model. Target-definition is something we own and need to manage updates for.

Reply:
This is not a blocker but a strong recommendtaion going foward.

Reply:
ok. Thanks.